src/Aspire.Hosting/ResourceBuilderExtensions.cs

```diff
@@ -2299,26 +2299,29 @@ public static IResourceBuilder<T> WithComputeEnvironment<T>(this IResourceBuilde
     /// Adds support for debugging the resource in VS Code when running in an extension host.
     /// </summary>
     /// <param name="builder">The resource builder.</param>
+    /// <param name="resourceType">The type of resource, such as 'project'.</param>
     /// <param name="projectPath">The path to the project file.</param>
     /// <param name="debugAdapterId">The debug adapter ID to use. Ie, coreclr</param>
     /// <param name="requiredExtensionId">The ID of the required VS Code extension. If specified, the extension must be installed for debugging to be enabled.</param>
     /// <param name="argsCallback">Optional callback to add or modify command line arguments when running in an extension host. Useful if the entrypoint is usually provided as an argument to the resource executable.</param>
-    [Experimental("ASPIREEXTENSION001")]
-    public static IResourceBuilder<T> WithVSCodeDebugSupport<T>(this IResourceBuilder<T> builder, string projectPath, string debugAdapterId, string? requiredExtensionId, Action<CommandLineArgsCallbackContext>? argsCallback = null) where T : IResource
+    /// <param name="launchConfiguration">Optional custom launch configuration.</param>
```

Comment:
If we are passing this, can we get rid of `resourceType` and `projectPath` parameters? They lauch configuration is already carrying this information